configDesktop.js

```diff
@@ -167,7 +167,10 @@ const tests = [
 	},
 	{
 		label: 'Test sticky header (#vector-2022, #logged-in, #scroll)',
-		path: '/wiki/Test'
+		path: '/wiki/Test',
+		viewports: [
+			VIEWPORT_DESKTOP
```

Comment:
For some reason, only the desktop wide test for this case was failing due to my refactoring. I think it has to do with the TOC pinning logic and how it ensures that the TOC is pinned by default. However, I dont know exactly how to fix this, so for simplicity sake I just removed the case, given the desktop wide is covered in other sticky header case